cmd/kata-local-ci

```diff
@@ -0,0 +1,271 @@
+    sys virt-builder $distro_name \
+         -o $disk --format qcow2 --size=$size \
+	 --root-password password:kata \
+	 "${distro_commands[@]}" \
+	 --mkdir /home/kata \
+	 --delete /.autorelabel \
+	 --ssh-inject "root:file:$key" \
+	 --install "git,rsync,curl,make,tar,tc" \
+         "${firstboot_cmds[@]}" \
+         "${final_firstboot_cmd[@]}" \
+	 --password  'kata:password:kata' \
+	 --hostname kata-$distro-ci
+    progress "Launching virt-install"
+    sys virt-install --import --name $VM_NAME  --memory $memory --vcpus $cpus \
+         --disk path=$disk,format=qcow2 \
+         --os-variant=$osvariant --noautoconsole $network
+
+    echo -n "Waiting for guest to get an IP address"
+    while ! sudo virsh domifaddr $VM_NAME | grep -q vnet ; do echo -n .; sleep 1; done
+
+    VM_IPADDR=$(sudo virsh domifaddr $VM_NAME | grep vnet | sed -e 's|^\s*vnet[0-9]*.*ipv4\s*\([0-9.]*\)/[0-9]*|\1|g')
+    echo "Got $VM_IPADDR"
+    progress "Guest IP address is $VM_IPADDR"
+
+    progress "Waiting for ssh to become available"
+    while ! ssh -o StrictHostKeyChecking=no -o UpdateHostKeys=no kata@$VM_IPADDR "mkdir -p ~/go/src/github.com/kata-containers"; do
+        sleep 1
+    done
+
+    progress "Initial rsync of kata-containers directory"
+    run rsync -av $directory/kata-containers kata@$VM_IPADDR:$vmdir/
+
+    progress "Initial rsync of tests directory"
+    run rsync -av $directory/tests kata@$VM_IPADDR:$vmdir/
+
+    progress "Configuring run in ~kata/ci_job_env"
+    cat > $kci/$distro-ci_job_env << EOF
+export GOPATH="\$HOME/go"
+export PATH="/usr/local/go/bin:\$GOPATH/bin:\$PATH"
+export CI="true"
+export CI_JOB="$job"
+pushd $vmdir/tests &>/dev/null
+source .ci/ci_job_flags.sh
+popd &>/dev/null
+EOF
+    run scp $kci/$distro-ci_job_env kata@$VM_IPADDR:ci_job_env
+
+    progress "Launching CI setup in $VM_NAME (this will take a while)"
+    ci_cmd "source ci_job_env; export CI_JOB=$job; cd $vmdir/tests; .ci/setup.sh"
+
+    cat > "$kci"/$distro-config <<EOF
+VM_NAME="$VM_NAME"
+VM_IPADDR=$VM_IPADDR
+VM_DISK=$disk
+EOF
+
+    progress "Taking snapshot of $VM_NAME"
+    ci_root_cmd "sync; sync; sync; sleep 3"
+    sys virsh snapshot-create-as $VM_NAME --name initial-snapshot --description "Initial snapshot of $VM_NAME after Kata CI setup"
+
+}
+
+if $install ; then
+    vm_install
+else
+    source "$kci"/$distro-config
+
+    if $revert ; then
+        progress "Reverting to initial snapshot of $VM_NAME"
+        sys virsh snapshot-revert $VM_NAME initial-snapshot
+    fi
+
+    progress "Updating rsync of kata-containers directory"
+    run rsync -av $directory/kata-containers kata@$VM_IPADDR:$vmdir/
+
+    progress "Updating rsync of tests directory"
+    run rsync -av $directory/kata-containers kata@$VM_IPADDR:$vmdir/
+fi
+
+progress "Cleanup previous runs"
+ci_root_cmd "swapoff -a" # Apparently, even if /etc/fstab has no swap entries, swap partitions are activated
+ci_root_cmd "rm -rf /etc/cni/net.d .kube"
```

Comment:
Should this be?:

```suggestion
ci_root_cmd "rm -rf /etc/cni/net.d $PWD/.kube"
```

Reply:
No, because this is a command sent over ssh. So `$PWD` would give me the wrong path. What I want is the user `.kube` file, and since `.ssh` brings me in the home directory, it does what I want.